docs/source/howto/level1-csd.rst

```diff
@@ -0,0 +1,312 @@
+
+**Downloading Landsat and Sentinel-2 Level-1 data from cloud services**
+
+This tutorial covers the process of querying and downloading Landsat and Sentinel-2 Level 1 data from cloud services (currently Google Cloud Storage).
+FORCE Level 1 CSD offers a high level of detail in requesting data and creates a consistent Level 1 data pool, as well as queue files that can be used for Level 2 processing.
+
+.. admonition:: Info
+
+   *This tutorial uses FORCE v. 3.5*
+
+FORCE Level 1 CSD makes use of the Landsat and Sentinel-2 metadata catalogues from Google Cloud Storage to find scenes matching the user's requirements and downloads them through `gsutil <https://cloud.google.com/storage/docs/gsutil>`_. A check for scenes already existing locally is performed and only new scenes are downloaded.
+
+
+Getting started
+---------------
+
+gsutil configuration
+""""""""""""""""""""
+
+Currently, Landsat data can be downloaded without a login (this will change at some point), but downloading Sentinel-2 data requires authentication.
+This is done through gsutil and only needs to be done once before the first run. The credentials will be stored in a file called ``.boto`` in the user's home directory (make sure to check the notes at the end of this document if you're running FORCE in Docker).
+
+Type the following command to start the authorization process
+
+.. code-block:: none
+
+    gsutil config
+
+
+Follow the on-screen instructions to authorize access to the Google Cloud Platform APIs. A project name is not required, just hit enter or make up a name when asked.
+
+Downloading the metadata catalogues
+"""""""""""""""""""""""""""""""""""
+
+The metadata catalogues contain metadata for all scenes stored on Google Cloud Storage. This includes all the information we need to find data that matches our needs - location, date, cloud cover estimates, sensor, etc...
+The ``-u | --update`` option will download the latest (compressed) catalogues and extract them to the metadata directory specified by the user.
+
+.. code-block:: none
+
+    force-level1-csd -u /data/Jakku/edc/level1/
+
+    Downloading Landsat metadata catalogue...
+    Extracting compressed Landsat metadata catalogue...
+
+    Downloading Sentinel-2 metadata catalogue...
+    Extracting compressed Sentinel-2 metadata catalogue...
+
+    Done. You can run this script without option -u to download data now.
+
+
+The ``-s | --sensor`` option may be used if only one of the two catalogues need to be downloaded / updated. Specifying one of the sensors of either family is sufficient (i.e., LT05 for Landsat or S2B for Sentinel-2).
+
+Parameterization
+----------------
+
+Basic syntax
+""""""""""""
+
+FORCE Level-1 CSD is parameterized with four mandatory arguments. These make up the core of the request and always need to be provided, while all remaining arguments are optional. The basic structure is visualized in the image below.
+
+.. figure:: img/tutorial-l1csd-syntax.jpg
```

Comment:
I know this does not matter for CSD, but in congruency with regular bash CMD syntax, it would be nice to have the optional parameters before the mandatory ones (as in the usage btw. "Usage: force-level1-csd [optional arguments] metadata-dir level-1-datapool queue aoi")